### DISCUSSION TITLE
Adding satellites with correct visual magnitude?

### DISCUSSION THREAD

**Discussion by cantino** [Stellarium/stellarium#2100] [Q&A] [answered]
I'm trying to visualize what a cluster of satellites in geostationary orbit would look like from the ground in the real world (if they only do E-W stationkeeping, then each satellite's ground trace will follow an analemma). If I add multiple satellites to stellarium, can I have them be rendered with correct magnitude, or will they all be rendered as an informational overlay? Would I need to add them as tiny fake planetary bodies or something?

**Comment by gzotti** [comment, marked as answer]
There is the Satellites plugin. Just add TLE. You can switch between symbols (icons) and a roughly realistic white-dot rendering which scales with apparent magnitude. But I cannot say how accurate the magnitude prediction is.

**Reply by alex-w** [reply]
Just add TLE in the Satellites plugin, but I should say, that not all satellites has standard magnitude or RCS data, so, not all satellites has accurate prediction of visual magnitude.

**Reply by cantino** [reply]
Thanks! I've been able to generate fake TLEs, but I don't know how to set the magnitude for a bunch of satellites.

**Reply by alex-w** [reply]
> Thanks! I've been able to generate fake TLEs, but I don't know how to set the magnitude for a bunch of satellites.

Do you have a standard magnitudes for satellites?

**Comment by cantino** [comment]
Yes I’d be happy if I could set them all to the same custom apparent
magnitude

**Reply by alex-w** [reply]
> Yes I’d be happy if I could set them all to the same custom apparent magnitude

You need to edit `satellites.json` file

**Reply by cantino** [reply]
Thanks!
